.custom-gcl.yml

```diff
@@ -0,0 +1,6 @@
+version:  v2.2.1
+name: golangci-kube-api-linter
+destination: ./bin
+plugins:
+- module: 'sigs.k8s.io/kube-api-linter'
+  version: 'main'
```

Comment:
I would recommend to pin a specific commit. While we are still in the "early days" we haven't yet committed to producing actual releases, and I'm concerned that if you track `main`, we might break you accidentally.

In paritcular, I know that we need to rework how `requiredfields` works after recent discussion with upstream API reviewers, so that one will probably be surprising if it suddenly starts reporting issues

Reply:
It's on the todo list, as both this project and Kubebuilder have expressed an interest in proper pinned releases. I need to get the required fields refactor I'm working on out of the way, and then creating an actual `cmd` that can be built and shipped is the next thing. The `cmd` will just be `golangci-lint` wrapping KAL, so the releases I expect will be tagged something like `golangci-lint-vA.B.C+kube-api-linter-vX.Y.Z`